dist/es2015/aurelia-binding.js

```diff
@@ -3078,51 +3078,53 @@ let CapturedHandlerEntry = class CapturedHandlerEntry {
     }
   }
 };
+let DelegateHandlerEntry = class DelegateHandlerEntry {
+  constructor(eventName, eventManager) {
+    this.eventName = eventName;
+    this.count = 0;
+    this.eventManager = eventManager;
+  }
 
+  handleDelegatedEvent(event) {
```

Comment:
```suggestion
  handleEvent(event) {
```
And then in `increment()`:
```ts
DOM.addEventListener(this.eventName, this, false);
```
and in `decrement()`:
```ts
DOM.removeEventListener(this.eventName, this, false);
```